content/blog/cloud-native-infrastructure-with-kubernetes-and-pulumi/index.md

```diff
@@ -131,23 +130,18 @@ entry for using Kubernetes with real components and reuse.
 We can't wait to hear what you think. To learn more or give it a try,
 please check out these resources:
 
-[Instal Pulumi]({{< ref "install" >}})
+[Instal Pulumi]({{< ref "/docs/reference/install.md" >}})
```

Comment:
Super nit: In some cases `ref` is being used (which emits full URL including the baseURL) and other cases `relref` is used (which is just a relative URL). Is there a reason why or just an oversight? It'd be nice to be consistent. I've used `relref` everywhere else on the docs site.